src/containers/Form/index.tsx

```diff
@@ -54,8 +55,14 @@ const Form = ({
         };
 
         setLoading(true);
-        const res = await axios.post(`/api/claim`, payload);
-        if (res.status === 201) {
+        const session = await LoadSession();
+        let res
```

Comment:
@sauramirez is this a good way or how should we handle checking how the user is logged in?